sopel_waifu/waifu.json

```
@@ -1453,37 +1506,112 @@
         "Yuika Mitsumine"
     ],
     "The Tatami Galaxy": [
-        "Akashi"
+        "Akashi",
+        "Hanuki Ryouko",
+        "Higuchi Keiko",
+        "Kaori"
     ],
     "To Aru Kagaku no Railgun": [
-        "Shirai Kuroko",
         "Saten Ruiko",
-        "Uiharu Kazari"
+        "Kiyama Harumi",
+        "Konori Mii",
+        "Juufuku Miho",
+        "Tessou Tsuduri",
+        "Wannai Kinuho",
+        "Awatsuki Maaya",
+        "Haruue Erii",
+        "Therestina Kihara Lifeline",
+        "Edasaki Banri",
+        "Kounoe",
+        "Hazamaya Kana",
+        "Nunotaba Shinobu",
+        "Frenda Seivelun",
+        "Mugino Shizuri",
+        "Takitsubo Rikou",
+        "Shokuhou Misaki",
+        "Kinuhata Saiai",
+        "Minori",
+        "Febri",
+        "Hokaze Junko",
+        "MISAKA 1"
+        "Kumokawa Seria",
+        "Xochitl",
+        "Taroumaru Seike",
+        "Yakumaru Itsuki",
+        "Iizumi Rita",
+        "Sakuragi Naruha",
+        "Dolly",
+        "Kouzaku Mitori",
+        "Kuriba Ryouko",
+        "Yumiya Rakko"
     ],
     "To Aru Majutsu no Index": [
-        "Index",
-        "Last Order",
+        "Index Librorum Prohibitorum",
         "Misaka \"Biribiri\" Mikoto",
-        "Misaka Imouto",
+        "Kanzaki Kaori",
         "Tsukuyomi Komoe",
+        "Kuroko Shirai",
         "Himegami Aisa",
+        "MISAKA 10032",
+        "Tsuchimikado Maika",
+        "Sasha Kreuzhev",
+        "Last Order",
+        "Kazakiri Hyouka",
+        "Itsuwa",
+        "Uiharu Kazari",
+        "Sherry Cromwell",
+        "Yomikawa Aiho",
+        "Yoshikawa Kikyou",
+        "Ryoukan",
+        "Laura Stewart",
+        "Kamijou Shiina",
+        "MISAKA 10031",
+        "MISAKA 19090",
         "Fukiyose Seiri",
-        "Mitsuko Kongo"
+        "Mitsuko Kongo",
+        "Vento of the Front",
+        "Oriana Thomson",
+        "Lidvia Lorenzetti",
+        "Lucia",
+        "Angelene",
+        "Agnese Sanctis",
+        "Orsola Aquinas",
+        "Misaka Misuzu",
+        "Musujime Awaki",
+        "Vasilisa",
+        "Uragami"
     ],
     "Tonari no Kaibutsu-kun": [
-        "Shizuku \"Mitti\" Mizutani",
-        "Asako Natsume"
+        "Mizutani \"Mitti\" Shizuku",
+        "Natsume Asako",
+        "Ooshima Chizuru",
+        "Miyama Yuu",
+        "Ninomiya Saeko",
+        "Yumeko"
     ],
     "Top wo Nerae!": [
-        "Takaya Noriko"
+        "Takaya Noriko",
+        "Amano Kazumi",
+        "Jung Freud",
+        "Kashiwara Reiko"
```

```suggestion
        "Takaya Noriko",
        "Amano Kazumi",
        "Jung Freud",
        "Kashiwara Reiko",
        "Linda Yamamoto",
        "Akai Takami",
        "Higuchi \"Akai Kimiko\" Kimiko"
```

Not sure the best approach to that last one. Some DBs show her as `Kimiko "Kimiko Akai" Higuchi` to make it clear that she's related to Akai Takami (parent of), but either way it looks awkward.

---

I think it's best to stick with the surname-given name approach in this case.

---

So, this?

```suggestion
        "Takaya Noriko",
        "Amano Kazumi",
        "Jung Freud",
        "Kashiwara Reiko",
        "Linda Yamamoto",
        "Akai Takami",
        "Higuchi Kimiko"
```

What I'd _prefer_ is the Japanese equivalent of "[née](https://en.wiktionary.org/wiki/n%C3%A9e)", but the small amount of time I could spend researching possibilities for that turned up nothing useful.